starters/expo-zustand-styled-component/src/screens/Home/Home.tsx

```diff
@@ -0,0 +1,33 @@
+import { Image, TouchableOpacity } from 'react-native';
+import React from 'react';
+import { SafeAreaViewStyled, RawStyled, TitleStyled, CountStyled } from './Home.styles';
+import Button from '../../components/Button';
+import { useCount } from './useCount';
+
+const Home = () => {
+  const count = useCount((state) => state.count);
+  const increment = useCount((state) => state.increment);
+  const decrement = useCount((state) => state.decrement);
+  const reset = useCount((state) => state.reset);
+
+  return (
+    <SafeAreaViewStyled>
+      <RawStyled>
+        <TouchableOpacity>
+          <Image source={require('../../../assets/about-icon.png')} />
```

Comment:
Out of curiosity, what is supposed to happen when the user clicks on the about icon?
Is there supposed to be a modal that pops open when information about the app?
Or is it purely decorative?